test/cql-pytest/test_describe.py

```diff
@@ -2567,3 +2568,90 @@ def aux(row: DescRow) -> DescRow:
             assert restore_stmts == res
         finally:
             cql.execute(f"DROP KEYSPACE IF EXISTS {ks}")
+
+### ===========================================================================
+
+# `varchar` is an alias for `text`. Scylla defaults to `text` and we test it here.
+# It's not very important, but it's better to have a test that detects changes
+# like that and verifies we handle that case as expected.
+def test_restore_udf_with_varchar(cql, test_keyspace):
```

Comment:
If I understand correctly, because this test uses Lua, it is necessarily scylla_only and won't work in Java - and it's missing the scylla_only fixture.
Since clearly I didn't review much of the code in test_describe.py (as I said above, I must have been sleeping under a rock), other tests may have forgotten this too.
Please run your cql-pytests on Cassandra too (test/cql-pytest/run-cassandra, see instructions in test/cql-pytest/README.md) and be aware if your tests work on Scylla only - and if so, is it deliberate (e.g., before UDF is different), or by mistake.

Reply:
Also, why is this test called test_restore_udf (!?) instead of test_desc_function? It tests "desc function" and not "restore" (which sounds related to backups).

Reply:
That's a good point, thank you. The test file won't run against Cassandra for a variety of reasons. I'll create an issue for that, but we can ignore it now. The tests I left in the new version are Scylla-only.